main.sh

```diff
@@ -188,6 +187,32 @@ ebpf() {
 	fi
 }
 
+yq_install() {
+	# Will only install yq if GRAFANA_ENABLE option is set to true
+	if is_set "$GRAFANA_ENABLE"; then
+		command -v yq >/dev/null 2>&1 && {
+			ok "yq is already installed"
+			return 0
+		}
+
+		GOOS="$(go env GOOS)"
```

Comment:
golang is not pre installed in this repo, hence when integration on a new created ec2 instance, this will fail. Is there any possible to use uname to replace goarch? or install yq without golang/ or jq?

Reply:
yeah - I can get it to install without a golang dep - let me update